### PR TITLE
fix(Message): fix message RTL display location error

### PR DESCRIPTION
**Pull request by Yuiai01**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/39244
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix message RTL display location error         |
| 🇨🇳 Chinese | 修复 message 组件设置 direction:rtl 时出现位置错误          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/message/style/index.tsx [hunk 1]
```diff
@@ -71,7 +71,6 @@ const genMessageStyle: GenerateStyle<MessageToken> = (token) => {
         ...resetComponent(token),
         position: 'fixed',
         top: marginXS,
-        insetInlineStart: 0, // affected by ltr or rtl
```
Comment (yoyo837) [inline components/message/style/index.tsx:74]: 删除了ltr不会有问题吗？

Comment (Yuiai01) [reply components/message/style/index.tsx:74]: > 删除了ltr不会有问题吗？

测试过，删了之后就正常了
image

Comment (afc163) [reply components/message/style/index.tsx:74]: 这个 Popover ... https://github.com/ant-design/ant-design/issues/39195

Comment (Yuiai01) [reply components/message/style/index.tsx:74]: > 这个 Popover ... #39195
/(ㄒoㄒ)/~~ 打开就挂着了，没去管

**Review by afc163 — APPROVED** [review]

File: components/message/style/index.tsx (hunk 1 above)
Comment (afc163) [reply components/message/style/index.tsx:74]: @Yuiai01 有空也可以顺手帮忙看看

Comment (Yuiai01) [reply components/message/style/index.tsx:74]: > 
嗯嗯

**Pull request merged**